packages/server/docs/openapi.yaml

```diff
@@ -0,0 +1,1933 @@
+      responses:
+        '204':
+          $ref: '#/components/responses/EmptyObject'
+        '401':
+          $ref: '#/components/responses/ErrorAuthentication'
+        '404':
+          $ref: '#/components/responses/ErrorNotFound'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resource/draft/delete/{id}:
+    parameters:
+      - $ref: '#/components/parameters/idPath'
+    post:
+      description: 'Delete a draft resource.'
+      summary: 'Delete draft resource'
+      tags:
+        - resource
+      operationId: resourcesDraftDeletePost
+      security:
+        - requireAdmin: []
+      requestBody:
+        $ref: '#/components/requestBodies/EmptyRequest'
+      responses:
+        '204':
+          $ref: '#/components/responses/EmptyObject'
+        '401':
+          $ref: '#/components/responses/ErrorAuthentication'
+        '404':
+          $ref: '#/components/responses/ErrorNotFound'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resource/draft/{id}:
+    parameters:
+      - $ref: '#/components/parameters/idPath'
+    get:
+      description: 'Get a resource draft.'
+      summary: 'Get a resource draft'
+      operationId: resourceDraftGet
+      tags:
+        - resource
+      responses:
+        '200':
+          description: Single Resource Draft - `createdBy` only available to admin users.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  draftResource:
+                    $ref: '#/components/schemas/ResourceDraft'
+        '404':
+          $ref: '#/components/responses/ErrorNotFound'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resources/drafts:
+    parameters:
+      - $ref: '#/components/parameters/includeDeletedQueryOptional'
+    get:
+      description: 'Get an array of draft resources.'
+      summary: 'Get draft resources.'
+      tags:
+        - resource
+      operationId: resourcesDraftsGet
+      responses:
+        '200':
+          description: Draft resources
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  draftResources:
+                    type: array
+                    items:
+                      $ref: '#/components/schemas/ResourceDraft'
+        '500':
+          $ref: '#/components/responses/ErrorUnknown'
+
+  /resources/drafts/mine:
+    get:
+      description: 'Get an array of resources.'
+      summary: 'Get resources.'
+      tags:
+        - resource
+      operationId: resourcesDraftsMineGet
+      security:
+        - requireLoggedIn: []
+      responses:
+        '200':
+          description: Resource Drafts
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  uncategorizedResources:
```

Comment:
This would be `draftResources` that are an array of drafts.
https://github.com/CodeForBoulder/upswyng/blob/master/packages/server/src/routes/api/resources/drafts/mine.ts#L38